README.md

```diff
@@ -26,14 +26,24 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] Mention two parts of Express that you learned about this week.
 
+Express server and express Routers
+
 - [ ] Describe Middleware?
 
+Middleware is something that the code "filters" through for things like authentication, etc.
+
 - [ ] Describe a Resource?
 
+A resource is something that is used as a basis.
```

Comment:
try being a bit more descriptive. I know, english isn't your first language but it's not worth your